File: application/src/main/java/com/study/platform/domain/comment/usecase/CreateCommentUseCase.java
package com.study.platform.domain.comment.usecase;

import com.study.platform.domain.comment.dto.request.CommentCreateRequest;
import com.study.platform.domain.comment.dto.response.CommentResponse;

import java.util.UUID;

public interface CreateCommentUseCase {
    CommentResponse execute(UUID userId, UUID postId, CommentCreateRequest request);
}

File: application/src/main/kotlin/com/study/platform/domain/comment/usecase/CreateCommentUseCase.kt
package com.study.platform.domain.comment.usecase

import com.study.platform.domain.comment.dto.request.CommentCreateRequest
import com.study.platform.domain.comment.dto.response.CommentResponse
import java.util.UUID

interface CreateCommentUseCase {
    fun execute(userId: UUID, postId: UUID, request: CommentCreateRequest): CommentResponse
}
